key.ui/src/main/java/de/uka/ilkd/key/util/CommandLine.java

```diff
@@ -189,6 +185,42 @@ protected void print(PrintStream ps, int descriptionCol) {
     public CommandLine() {
     }
 
+    private String usedSubCommand = "";
+
+    private final Map<String, CommandLine> subcommands = new HashMap<>();
+
+    public CommandLine addSubCommand(String name) {
```

Comment:
Can you add some documentation for this, esp. what the return value is?

Reply:
Done.